tools/monitoring/mysql/database.go

```diff
@@ -139,6 +139,21 @@ func (db *DB) AddAlert(errorPattern string) error {
 	return err
 }
 
+// DeleteAlert deletes a row (alert) from the Alerts table
+func (db *DB) DeleteAlert(errorPattern string) error {
+	stmt, err := db.Prepare(`
+				DELETE FROM Alerts
+				WHERE ErrorPattern = ?`)
+	defer stmt.Close()
+
+	if err != nil {
+		return err
+	}
+
+	_, err = stmt.Exec(errorPattern)
```

Comment:
should we also check that only one row was affected?

Reply:
Updated